setup/README.md

```diff
@@ -38,6 +38,8 @@ git clone https://github.com/sse-secure-systems/connaisseur.git
 cd connaisseur
 ```
 
+> The default of what you will install when following this guide will always be the latest release of Connaisseur. If you want to run Connaisseur with the most recent changes that haven't yet made it into a release, you will have to build an image yourself with `make docker`.
```

Comment:
would refer to the contributing guide here, as it sketches how to setup your own env: https://github.com/sse-secure-systems/connaisseur/blob/master/CONTRIBUTING.md#setup-the-environment